samples/CMakeLists.txt

```diff
@@ -13,39 +13,41 @@ else()
 endif()
 set(CMAKE_BUILD_WITH_INSTALL_RPATH ON)
 
+MESSAGE(STATUS ${PATH_TO_DATA})
+
 add_custom_target(copy_data_dir
-    COMMENT "Copy project data directory to the runtime folder."
-    COMMAND cmake -E copy_directory
-        ${CMAKE_SOURCE_DIR}/../data
+        COMMENT "Copy project data directory to the runtime folder."
+        COMMAND cmake -E copy_directory
+        ${PATH_TO_DATA}
         ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/data
-)
+        )
 add_custom_target(copy_font
-    COMMENT "Copy terminal.png to the runtime folder."
-    COMMAND cmake -E copy_if_different
-        ${CMAKE_SOURCE_DIR}/../terminal.png
+        COMMENT "Copy terminal.png to the runtime folder."
+        COMMAND cmake -E copy_if_different
+        ${PATH_TO_TERMINAL_PNG}
         ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/terminal.png
-)
+        )
```

Comment:
What happened to the indentation here?  The variables had a second indentation because they were arguments to `COMMAND`.  This suddenly breaks both the one true brace style and the 4 spaces per indent.

Reply:
My editor settings do the formatting of code when it is sent in the Commit, so I never notice when it does. I'm sorry I messed up the indent.